scripts/entrypoint.sh

```diff
@@ -75,10 +75,28 @@ if [ -z "$POSTGRES_ROLE_EXISTS" ]; then
 EOSQL
 fi
 
-# We need to restart the server for the changes above
-# to be reflected
+# We need to restart the server for the changes above to be reflected
 pg_ctl restart
 
+# We send basic, anonymous deployment events to PostHog to help us understand
+# how many people are using the project and to track deployment success. We
+# only do this if TELEMETRY is set to "true", and only do it once per deployment
+if [[ ${TELEMETRY:-} == "true" ]]; then
+  curl -s -L --header "Content-Type: application/json" -d '{
+    "api_key": "'"$POSTHOG_API_KEY"'",
+    "event": "ParadeDB Deployment",
+    "distinct_id": "'"$(uuidgen)"'",
+    "properties": {
+      "commit_sha": "'"${COMMIT_SHA:-}"'"
+    }
+  }' "$POSTHOG_HOST/capture/" > /dev/null
+fi
+
+# Mark telemetry as handled so we don't try to send it again when
+# initializing our PostgreSQL extensions. We use a file for IPC
+# between this script and our PostgreSQL extensions
+echo "true" > /tmp/telemetry
+
 echo "PostgreSQL is up - installing extensions..."
```

Comment:
This looks great. I'm not totally clear from reading what the default for TELEMETRY is. Is this opt-in or opt-out?

This is good documentation for the entrypoint file, but we should probably repeat this in the README and be clear about what will happen by default.

Reply:
Agreed, will clarify